web/app/Application.js

```diff
@@ -164,15 +164,24 @@ Ext.define('Traccar.Application', {
         }
     },
 
-    showError: function (error) {
+    showError: function (error, success) {
```

Comment:
It doesn't make sense to use it for successful calls. I suggest to use `Ext.Msg.alert` directly.